Magazin/MainWindow.xaml

```diff
@@ -0,0 +1,49 @@
+﻿<Window x:Class="Magazin.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Magazin"
+        xmlns:views="clr-namespace:Magazin.Views"
+        xmlns:viewModels="clr-namespace:Magazin.ViewModels"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800">
+    <Window.DataContext>
+        <viewModels:MainWindowViewModel/>
+    </Window.DataContext>
+    <Window.Resources>
+        <DataTemplate x:Key="carTemplate">
+            <StackPanel Orientation="Horizontal">
+                <Image Margin="5" Width="80" Height="80" Source="{Binding Path=Picture}"></Image>
+                <TextBlock Margin="5" Text="{Binding Path=Brand}"></TextBlock>
```

Comment:
From my point of view it is better to use single tags where possible:
<TextBlock Margin="5" Text="{Binding Path=Brand}" />
instead of
<TextBlock Margin="5" Text="{Binding Path=Brand}"></TextBlock>